src/Npgsql/NpgsqlSpanStream.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Npgsql
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public abstract class NpgsqlSpanStream : NpgsqlStream
+    {
+        #region Constructor
```

Comment:
Regions is a good thing only in case when there are a lot of code. Otherwise, trey bring nothing except additional lines of code.

Reply:
Removed.